scripts/manual-diff-check.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * To execute this script, run the following row in the root directory of this project.
+ * Please add a valid github token and replace the PR number.
+ * rm -rf dist/screenshots && tsx scripts/manual-diff-check.ts --pr=3131 --github-token=<YOUR-TOKEN> && cd dist/screenshots && unzip screenshots.zip && cd ../.. && yarn start:visual-regression-app
+ */
+
+import { createWriteStream, existsSync, mkdirSync } from 'fs';
+import { Readable } from 'node:stream';
+import { finished } from 'node:stream/promises';
+import { parseArgs } from 'node:util';
+
+import { distDir } from '../tools/vite/index.js';
+
+const { values: cliArgs } = parseArgs({
+  strict: false,
+  options: {
+    pr: { type: 'string' },
+    'github-token': { type: 'string' },
+  },
+});
+
+async function downloadArtifact(prNumber: string): Promise<void> {
+  const prDetailsResponse = await fetch(
+    `https://api.github.com/repos/sbb-design-systems/lyne-components/pulls/${prNumber}`,
+  );
+
+  const prDetails = (await prDetailsResponse.json()) as any;
```

Comment:
Nitpick; Maybe create a mini interface of the data you need (e.g. just with head in this case)